source/update.rst

```diff
@@ -174,86 +177,87 @@ Manual update
 #. The update process is successfully finished when all Pods have been
    restarted.
 
-.. _operator-update-smartupdates:
+.. _operator-update:
 
-Upgrading Percona XtraDB Cluster
---------------------------------
+Upgrading the Operator
```

Comment:
The operator update instruction should have only the following points:
1. Update the Custom Resource Definition...
2. Now you should apply a patch to your deployment...
3. Now you should apply a patch to your deployment, supplying necessary image name  - _we need to rewrite it because  we change only one image for operator and it does not depend on version of PXC._
4. get status of operator `kubectl get deployments percona-xtradb-cluster-operator`

Manual upgrade and Semi-automatic upgrade are connected with the old pxc cluster update which was before 1.6.0 but not operator's update.